packages/flutter/lib/src/material/color_scheme.dart

```diff
@@ -749,18 +749,18 @@ class ColorScheme with Diagnosticable {
   /// ),
   /// ```
   /// {@end-tool}
-  const ColorScheme.highContrastLight({
+  ColorScheme.highContrastLight({
```

Comment:
Some where in the document should mention this will prioritize system colors

Reply:
Yep, once we all agree on the code changes, I'll go through docs and update. I didn't want to change the docs too early in the process and keep going back to change them multiple times.